src/routes/Home.js

```diff
@@ -5,39 +5,80 @@ import {
   Paneset,
   Pane,
 } from '@folio/stripes-components';
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  DataFetcher,
+  createUrl,
+} from '@folio/stripes-data-transfer-components';
 
 import {
   QueryFileUploader,
   Jobs,
   JobLogsContainer,
 } from '../components';
+import { DataFetcherContext } from '../contexts/DataFetcherContext';
+import { JOB_EXECUTION_STATUSES } from '../utils/constants';
+
+const jobsUrl = createUrl('data-export/jobExecutions', {
+  query: `status=(${JOB_EXECUTION_STATUSES.IN_PROGRESS})`,
+  limit: 50,
+}, false);
+
+const logsUrl = createUrl('data-export/jobExecutions', {
+  query: `status=(${JOB_EXECUTION_STATUSES.SUCCESS})`,
+  limit: 25,
+}, false);
 
-export default function Home() {
+function Home(props) {
   return (
     <Paneset>
-      <Pane
-        data-test-jobs-pane
-        defaultWidth="320px"
-        paneTitle={(
-          <span data-test-title>
-            <FormattedMessage id="ui-data-export.jobsPaneTitle" />
-          </span>
-        )}
+      <DataFetcher
+        FetcherContext={DataFetcherContext}
+        resources={props.resources}
+        mutator={props.mutator}
+        resourcesMappingPath="records.0.jobExecutions"
       >
-        <QueryFileUploader />
-        <Jobs />
-      </Pane>
-      <Pane
-        data-test-logs-pane
-        defaultWidth="fill"
-        paneTitle={(
-          <span data-test-title>
-            <FormattedMessage id="ui-data-export.logsPaneTitle" />
-          </span>
-        )}
-      >
-        <JobLogsContainer />
-      </Pane>
+        <Pane
+          data-test-jobs-pane
+          defaultWidth="320px"
+          paneTitle={(
+            <span data-test-title>
+              <FormattedMessage id="ui-data-export.jobsPaneTitle" />
+            </span>
+          )}
+        >
+          <QueryFileUploader />
+          <Jobs />
+        </Pane>
+        <Pane
+          data-test-logs-pane
+          defaultWidth="fill"
+          paneTitle={(
+            <span data-test-title>
+              <FormattedMessage id="ui-data-export.logsPaneTitle" />
+            </span>
+          )}
+        >
+          <JobLogsContainer />
+        </Pane>
+      </DataFetcher>
     </Paneset>
   );
 }
+
+Home.manifest = Object.freeze({
+  jobs: {
+    type: 'okapi',
+    path: jobsUrl,
```

Comment:
It seems, we can move jobsURL and logsURL closer to their usage. What do you think about it?

Reply:
Sounds good.